python/ray/tests/test_usage_stats.py

```diff
@@ -125,6 +125,8 @@ def reset_ray_version_commit():
 def test_get_extra_usage_tags_to_report(
     monkeypatch, call_ray_start, reset_usage_stats, ray_client, gcs_storage_type
 ):
+    if os.environ.get("RAY_MINIMAL") == "1" and ray_client:
```

Comment:
Add this to the test definition? 

```
@pytest.skipif(...)
def test_get_extra_usage_tags_to_report(...)
```

Reply:
how do we conditionally skip the test with `skipif`? I tried various ways and this approach is the most understandable one to me.